README.md

```diff
@@ -1,11 +1,294 @@
+accrue progress when shopping at the stores.
+
+## Integration Testing
+
+The SDK will provide an option to run in test mode. In that mode, the web view
+and the API will point to a dev environment where the developer will be able to
+test the integration without affecting production data (more details when SDK
+is delivered).
+
+## Installation
+
+#### Via Cocoapods
+
+The ThanxSDK pod lives in [Thanx's Spec repo](https://github.com/thanx/Specs),
+for this reason, the first thing you need to add to your **Podfile** is the repo
+url as a new source:
+
+```ruby
+# Podfile
+source 'https://github.com/thanx/Specs.git'
+```
+
+Then you can include the *ThanxSDK* pod as one of your dependencies:
+
+```ruby
+# Podfile
+source 'https://github.com/thanx/Specs.git'
+
+target 'Your App' do
+  use_frameworks!
+
+  # Thanx SDK pod
+  pod 'ThanxSDK'
+
+  # ... Other dependencies ...
+end
+```
+
+The SDK uses the camera and photo library in order to upload receipts so make
+sure to add the following in your plist:
+
+```
+<key>NSCameraUsageDescription</key>
+<string>This app requests access to your camera and photo library to allow you to easily add credit cards (by reading them with your camera) as well as upload photos of receipts.</string>
+<key>NSPhotoLibraryUsageDescription</key>
+<string>This app requests access to your camera and photo library to allow you to easily add credit cards (by reading them with your camera) as well as upload photos of receipts.</string>
+```
+
+## Initialization
+
+You should initialize the SDK in the **AppDelegate** file of your project:
+
+1. Import the SDK at the top of the file: **import ThanxSDK**
+2. Call **Thanx.initialize** on the
+  **application:didFinishLaunchingWithOptions** method
+
+**Swift**:
+
+```swift
+// AppDelegate.swift
+import ThanxSDK
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    Thanx.initialize(clientId: "YOUR_CLIENT_ID", clientSecret: "YOUR_CLIENT_SECRET", debug: true)
+    return true
+  }
+}
+```
+
+The Thanx SDK is now initialized and ready for use.
+
+<aside class="notice">
+  Make sure to set <strong>debug: false</strong> for production builds.
+</aside>
+
+## Launching the Thanx experience
+
+The Thanx SDK provides the **WebViewController** with the full Thanx mobile
+experience. The UIViewController can be presented in any way. Example:
+
+**Swift**:
+
+```swift
+// UIViewController.swift
+# Present it as a full screen modal
+let thanxWebViewController = Thanx.WebViewController(showToolbar: true)
+present(thanxWebViewController, animated: true)
+
+# Present it as a part of a UINavigationController
+let thanxWebViewController = Thanx.WebViewController(showToolbar: false)
+navigationController?.pushViewcontroller(thanxWebViewController, animated: true)
+```
+
+## Push Notifications
```

Comment:
are we sure these work?

Reply:
Yup - just tested with Lou's app using the SDK:
![img_2961](https://user-images.githubusercontent.com/885841/51645805-b955b480-1f2a-11e9-8181-faeca058e002.PNG)